medusa/server/api/v2/config.py

```diff
@@ -596,72 +599,74 @@ def data_main():
 
-        section_data['values'] = {}
-        section_data['values']['na'] = common.Quality.NA
-        section_data['values']['unknown'] = common.Quality.UNKNOWN
-        section_data['values']['sdtv'] = common.Quality.SDTV
-        section_data['values']['sddvd'] = common.Quality.SDDVD
-        section_data['values']['hdtv'] = common.Quality.HDTV
-        section_data['values']['rawhdtv'] = common.Quality.RAWHDTV
-        section_data['values']['fullhdtv'] = common.Quality.FULLHDTV
-        section_data['values']['hdwebdl'] = common.Quality.HDWEBDL
-        section_data['values']['fullhdwebdl'] = common.Quality.FULLHDWEBDL
-        section_data['values']['hdbluray'] = common.Quality.HDBLURAY
-        section_data['values']['fullhdbluray'] = common.Quality.FULLHDBLURAY
-        section_data['values']['uhd4ktv'] = common.Quality.UHD_4K_TV
-        section_data['values']['uhd4kwebdl'] = common.Quality.UHD_4K_WEBDL
-        section_data['values']['uhd4kbluray'] = common.Quality.UHD_4K_BLURAY
-        section_data['values']['uhd8ktv'] = common.Quality.UHD_8K_TV
-        section_data['values']['uhd8kwebdl'] = common.Quality.UHD_8K_WEBDL
-        section_data['values']['uhd8kbluray'] = common.Quality.UHD_8K_BLURAY
-
-        section_data['anySets'] = {}
-        section_data['anySets']['anyhdtv'] = common.Quality.ANYHDTV
-        section_data['anySets']['anywebdl'] = common.Quality.ANYWEBDL
-        section_data['anySets']['anybluray'] = common.Quality.ANYBLURAY
-
-        section_data['presets'] = {}
-        section_data['presets']['any'] = common.ANY
-        section_data['presets']['sd'] = common.SD
-        section_data['presets']['hd'] = common.HD
-        section_data['presets']['hd720p'] = common.HD720p
-        section_data['presets']['hd1080p'] = common.HD1080p
-        section_data['presets']['uhd'] = common.UHD
-        section_data['presets']['uhd4k'] = common.UHD_4K
-        section_data['presets']['uhd8k'] = common.UHD_8K
-
-        section_data['strings'] = {}
-        section_data['strings']['values'] = common.Quality.qualityStrings
-        section_data['strings']['anySets'] = common.Quality.combinedQualityStrings
-        section_data['strings']['presets'] = common.qualityPresetStrings
-        section_data['strings']['cssClass'] = common.Quality.cssClassStrings
+        section_data['qualities'] = {}
+
+        # Get key names from `common.Quality` class (for the quality values and any-sets)
+        quality_key_map = {
+            value: text_type(key).lower().replace('_', '')
+            for (key, value)
+            in iteritems(common.Quality.__dict__)
+            if isinstance(value, int) and not key.startswith('__')
+        }
+
+        def make_quality(value, name, key=None):
+            return {
+                'value': value,
+                'key': key or quality_key_map.get(value),
+                'name': name,
+                'cssClass': common.Quality.cssClassStrings.get(value) if key is None else name,
```

Comment:
I think the cssClassStrings is a bit of unwanted legacy. Can't we refactor this a little and use `name` for the class?

Reply:
Hmm. Some `name`s have spaces in them. Maybe I'll use the `key`s and update the CSS for it.
I'll look into it later.
Thanks!

Reply:
Agree, key is better